lib/broccoli/glimmer-app.ts

```diff
@@ -417,27 +415,19 @@ Please run the following to resolve this warning:
   }
 
   private cssTree() {
-    // should really make SASS support to be opt-in, so that
-    // we can properly honor the `GlimmerAppOptions.trees.src`
-    // abstraction here, but for now we still require `src` to be a
-    // "real" path on disk that we check
     let stylesPath = path.join(resolveLocal(this.project.root, 'src'), 'ui', 'styles');
```

Comment:
I'd like to update `buildTrees` to emit a `styles` tree (in addition to `src` and `nodeModules`). And allow `GlimmerAppOptions.trees.styles` to be passed in (to override the default styles tree selection in `buildTrees`), then `this.trees.styles` would be used here instead. 

This is what the comments above were referring to.